examples/app/bokeh-decision-tree/templates/index.html

```diff
@@ -0,0 +1,52 @@
+{% extends base %}
+
+{% block title %}Decision Tree Visualizer{% endblock %}
+
+{% block postamble %}
+  <style>
+    {% include 'styles.css' %}
+  </style>
+{% endblock %}
+
+{% block contents %}
+  <div class="content" align="center">
+    <h1>Decision Tree Visualizer</h1>
+
+    <div style="margin-left: auto; margin-right: auto; width: 40em; font-family: Arial; text-align: left">
+        <p>ID3 algorithm is used to create the decision tree below using gini-index.
+        For positioning of the tree, <a href="https://link.springer.com/chapter/10.1007/3-540-36151-0_32">Buchheim's Paper</a>
+        was implemented.</p>
+        <p>For better understanding of the decision tree concept, ability to change the root attribute, opt-in and opt-out attributes have been included.</p>
+        <br> Datasets used in the tree:
+        <a href="https://archive.ics.uci.edu/ml/datasets/lenses"> Lenses</a>
+        <a> - </a>
+        <a href="https://archive.ics.uci.edu/ml/datasets/mushroom"> Mushrooms</a>
+    </div>
+    {{ super() }}
+
+    <div id="About" style="text-align:center; margin-left: auto; margin-right: auto; display: block">
+        <h2 style="color:dimgrey; text-align:center" >About</h2>
+        <hr style="width: 10%; margin-left: auto; margin-right: auto;">
+        <a href="http://analyticslab.sabanciuniv.edu/">
+            <img src="https://pbs.twimg.com/profile_images/730662586992230400/lPApNiEi_400x400.jpg" style="width: 10%; height: 10%; margin-left: auto; margin-right: auto;">
+        </a>
+        <br>
+
+        <a href="http://www.linkedin.com/in/onur-yigit-arpali">
+            <img src="bokeh-decision-tree/static/images/onur.jpg" width="100">
+        </a>
+        <a href="https://www.linkedin.com/in/evrimo/">
+            <img src="bokeh-decision-tree/static/images/evrim.jpg" width="100">
+        </a>
+        <a href="https://www.linkedin.com/in/ziyaerkoc/">
+            <img src="bokeh-decision-tree/static/images/ziya.jpg" width="100">
+        </a>
+        <a href="https://www.linkedin.com/in/artun-sar%C4%B1oglu-7b5107159/">
+            <img src="bokeh-decision-tree/static/images/artun_gs.jpg" width="100">
+        </a>
+        <a href="mailto:y.umutciftci@gmail.com">
+            <img src="bokeh-decision-tree/static/images/umut.jpeg" width="120">
+        </a>
+    </div>

```

Comment:
Not sure, what these links and images are for?

Reply:
These are images of ours containing links to our mails and linked-in profiles in the About section of the web page.

Reply:
I don't think this kind of promotion is appropriate for examples in the repo, there are no other examples that do this. If you'd like to have an example with materials for self-promotion they should go in a separate repo, again we would be happy to link to from the docs.

Reply:
Let me clear. You are suggesting that in Bokeh's web-site you will provide link to our web-page served by us containing this example instead of contributing to _examples/app_ and being served by you. In that way, our source code will be in another repository.